themes/salix/layouts/shortcodes/contact.html

```diff
@@ -0,0 +1,18 @@
+<section class="contact">
+	<div>
+		<h2 id="contact">Contact</h2>
+		<div>
+			<p>
+				{{ .Inner }}
+			</p>
+			<a href="mailto:{{ .Site.Params.contact.email }}">
+				{{ .Site.Params.contact.email }}
```

Comment:
I know I'm being super nitpicky here in terms of best practices, but I think *this* should be the shortcode (ie. "email-pillbox" or something), not the entire section including some of its content. Ie. let's just make this a section with an ID, in the CSS apply the pigeon as a background on that section ID, and in the markdown content add the "h2" and the email shortcode.

Reply:
> I like the pill box; it feels a bit out of place because the rest of the site is kind of square, but the idea is really cool (and I'm not against leaving it as is, it looks good and I am certainly not the design person in the co-op).

yeah it was a bit of a stab in the dark, I should probably do a bit of visual research to see how other people have solved this issue then makes some variations in Figma. I just wanted something that would make it accessible while changing the design as little as possible for the mean time.

Reply:
I'll follow this up, I'm concerned about being able to accurately place the image but I'll have an experiment and see if I can make it work.

Reply:
Generally speaking I think web design needs to operate within the constraints of web development. If we can't match the design exactly 1:1 that's fine. Changes will likely have to be made for eg. accessibility or to follow best practices and that's okay. Absolute positioning isn't one of the things we can do with web design, so might as well just turn off repeat, position it as well as we can, and call it a day IMO :)